components/drata/sources/failed-monitor/failed-monitor.mjs

```diff
@@ -0,0 +1,101 @@
+import drata from "../../drata.app.mjs";
+import { DEFAULT_POLLING_SOURCE_TIMER_INTERVAL } from "@pipedream/platform";
+
+const docsLink = "https://developers.drata.com/docs/openapi/reference/operation/MonitorsPublicController_listMonitors/";
+
+export default {
+  key: "drata-failed-monitor",
+  name: "Failed Monitor",
+  description: `Emit new event whenever a monitor fails. [See docs here](${docsLink}).`,
```

Comment:
```suggestion
  description: `Emit a new event whenever a monitor fails. [See the docs here](${docsLink}).`,
```

Reply:
Changed 'new event' to 'a new event' and 'docs' to 'the docs' to follow Google's Developer docs style guide.